src/views/Search/Record.vue

```diff
@@ -34,7 +56,13 @@
             return {
                 title: this.getTitle()
             }
-        }
+        },
+        mounted: function () {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The "mounted" property should be above the "methods" property on line 39.](https://app.codacy.com/gh/FAIRsharing/fairsharing.github.io/pullRequest?prid=4893952)